locale/en-US/standalone.properties

```diff
@@ -4,8 +4,6 @@
 
```

Comment:
While removing from standalone, `rooms_default_room_name_template` and `vendorShortname` seems to be unused. Added by 1e825246 and 57cffe46